internal/cmd/policy/reorderrules/reorderrules.go

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package reorderrules
+
+import (
+	"github.com/gittuf/gittuf/internal/cmd/common"
+	"github.com/gittuf/gittuf/internal/cmd/policy/persistent"
+	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	p          *persistent.Options
+	policyName string
+	ruleNames  []string
+}
+
+func (o *options) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&o.policyName,
+		"policy-name",
+		"targets",
+		"name of policy file to reorder rules in",
+	)
+
+	cmd.Flags().StringSliceVar(
+		&o.ruleNames,
+		"rule-name",
+		[]string{},
+		"a space-separated list of rule names, in the new order that they should appear in, from first to last",
+	)
+	cmd.MarkFlagRequired("rule-name") //nolint:errcheck
+}
+
+func (o *options) Run(cmd *cobra.Command, args []string) error {
+	o.ruleNames = append(o.ruleNames, args...)
```

Comment:
I missed this line before: what's this for?

Reply:
Yes, it ensures `--rule-name` works either way

Reply:
hm, is there anything that stops a rule name from having a space in it? This would break those cases (compared to `--rule-name "rule 1" --rule-name "rule 2"` I think.

cc @patzielinski

Reply:
If we use multiple quotes following `rule-name`, it can still parse those with space in them. Though we probably should specify that to users in the doc

<img width="1338" alt="Screenshot 2024-09-06 at 4 45 34 PM" src="https://github.com/user-attachments/assets/2a1add5b-0ff5-4c38-b0a0-31fef9cb6f88">
<img width="1346" alt="Screenshot 2024-09-06 at 4 45 52 PM" src="https://github.com/user-attachments/assets/7a900582-1c85-4b71-a10c-3be0b9d49711">

Reply:
thinking about this some more, do we even need `--rule-name` then? Can we just use `args`?

Reply:
Do you mean the rule names goes directly as `args`? I suppose we can still keep `--rule-name`, just to signal it is the start of specifying the new order

Reply:
No, I think we could just make that clear in the docs for the subcommand. Having a flag can be confusing if we really want them to pass in args. Let's get rid of the flag and use args only, and make that clear in the command docs, thanks!

Reply:
Gocha, I'll remove the flag and update the docs